nibabel/elemcont.py

```diff
@@ -0,0 +1,188 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+'''Containers for storing "elements" which have both a core data value as well
+as some additional meta data. When indexing into these containers it is this
+core value that is returned, which allows for much cleaner and more readable
+access to nested structures.
+
+Each object stored in these containers must have an attribute `value` which
+provides the core data value for the element. To get the element object itself
+the `get_elem` method must be used.
+'''
+from collections import MutableMapping, MutableSequence
+
+from .externals import OrderedDict
+from .externals.six import iteritems
+
+
+class Elem(object):
```

Comment:
Maybe 'MetaElem'? I think it helps make the difference between 'elements' and 'values' clear.